client/landing/subscriptions/components/tab-views/sites/sites.tsx

```diff
@@ -63,7 +83,7 @@ const Sites = () => {
 				<Notice type={ NoticeType.Warning }>
 					{ translate( 'Sorry, no sites match {{italic}}%s.{{/italic}}', {
 						components: { italic: <i /> },
-						args: searchTerm,
+						args: searchTerm || filterOption,
```

Comment:
I would argue that we need a new copy for this. The copy should talk about no sites matching the current search term (if any) **and** the current filter applied (if any).

@saygunnyc @yansern @michaelpick